packages/mdl-ripple/animation.js

```diff
@@ -2,13 +2,31 @@
  * Create a sequential animation/transition from a list of class names.
  */
 export class SequentialAnimation {
```

Comment:
since this is the only class in this file, how about `export default class SequentialAnimation`?